.github/workflows/docker-images.yml

```diff
@@ -14,8 +14,8 @@ on:
           - info
           - warning
           - debug
-#  pull_request: #for debugging purposes
-#    branches: [ "master" ]
+  pull_request: #for debugging purposes
+    branches: [ "master" ]
```

Comment:
I suggest that you add
https://github.com/k2-fsa/icefall/blob/master/.github/workflows/build-cpu-docker.yml#L3

After merging it into the master, you will see something below:

![Screenshot 2024-09-11 at 16 58 26](https://github.com/user-attachments/assets/64381af2-b0c4-472a-a69d-29f539b29df0)



You can trigger the actions manually by clicking, without pushing anything to github.